indigo/distribution.yaml

```diff
@@ -8399,6 +8399,23 @@ repositories:
       type: git
       url: https://github.com/silviomaeta/plot_util.git
       version: master
+  plotjuggler:
+    doc:
+      type: git
+      url: https://github.com/facontidavide/PlotJuggler.git
+      version: master
+    release:
+      packages:
+      - plotjuggler
```

Comment:
The second sentence of https://github.com/ros/rosdistro/pull/13670#discussion_r97373216 still applies. The `packages` entry needs to be removed since the only contained package name matches the repository name from line 8402 now.